features/validation_requests/managing_validation_requests.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Managing validation requests
+  Background:
+    Given I am logged in as an assessor
+    And a new planning application
+    And I view the planning application
+    When I press "Validate application"
+    And I press "Request validation changes"
+    And I create a new document validation request for a "Picture of the dog" because "it would be nice"
+    And I create a new document validation request for a "Picture of the cat" because "it would also be nice"
+
+  Scenario: As an assessor all my validation requests are initially not sent
```

Comment:
🤦‍♀️  My misreading. Because I'd stuck a comment in after line 3 that bisected the background, I was reading lines 4-9 as if they were a scenario rather than the background, and I thought: "why has he put the setup into a scenario?" Please mark as resolved and forget I said that